.github/workflows/publish.yml

```diff
@@ -110,7 +110,8 @@ jobs:
       if: github.ref != 'refs/heads/stable'
       run: |
         cd node
-        npm version --no-git-tag-version ${{ env.VERSION }}
+        export JS_LIBRARY_VERSION=$(npm run --silent version)
```

Comment:
JS_LIBRARY_VERSION will always match the version specified in package.json. For non-stable versions, we should add a suffix to the version name